test_scripts/pypeit_tests.py

```diff
@@ -200,6 +200,48 @@ def build_command_line(self):
                 self.setup.instr, '-c all', '-o', '--output_path', self.setup.rdxdir]
 
 
+class PypeItCalibsTest(PypeItTest):
```

Comment:
I don't like how this copies a lot of code from `PypeItReduceTest`. I think it would be better to either subclass `PypeItReduceTest` and override build_command_line, or add an argument to `PypeItReduceTest.init` that causes it to add the "-c" in build_command_line.  The second option might be better with the "--calib_only" argument (See my comments at the end)